kratos/python/add_mesh_to_python.cpp

```diff
@@ -466,6 +466,7 @@ void  AddMeshToPython(pybind11::module& m)
     .def("SetValue", SetValueHelperFunction< Element, Variable< std::string > >)
     .def("GetValue", GetValueHelperFunction< Element, Variable< std::string > >)
 
+    .def("PointsNumber", [](Element& rThis){return rThis.GetGeometry().PointsNumber();})
```

Comment:
This was my first thought. However, all the geometry accessible methods are masked throught the condition and element. I've the feeling that this was done in the past to ease the usage from Python. Hence, I decided to do it like this to be consistent with the rest.

Reply:
Ever since we added the exposure of the Geometry for Elememts&Conditions we have added new things there

Afaik it is quite complete by now, hence I would go the opposite direction and deprecate the old style interfaces. Then the usage would be consistent from C++ and python

Reply:
Makes complete sense for me. I'll do it that way then.